care/facility/api/serializers/patient.py

```diff
@@ -453,10 +454,20 @@ def create(self, validated_data):
 
     def save(self, **kwargs):
         self.instance.facility = self.validated_data["facility"]
-        PatientConsultation.objects.filter(
+        consultation = PatientConsultation.objects.filter(
```

Comment:
can you cover this statement and the next in a transaction?

Reply:
Done.